po/cs.po

```diff
@@ -0,0 +1,748 @@
+msgid "Show color names from the w3c basic color keyword set"
+msgstr "Zobrazit názvy barev ze základní w3c sady klíčových slov barev"
+
+#: src/widgets/preferences_window.rs:177
+msgctxt ""
+"Name of the extended color keyword set from https://www.w3.org/TR/css-"
+"color-3/#svg-color"
+msgid "Extended"
+msgstr "Rozšířené"
+
+#: src/widgets/preferences_window.rs:179
+msgid "Show color names from the w3c extended color keyword  set"
+msgstr "Zobrazit názvy barev z rozšířené w3c sady klíčových slov barev"
+
+#: src/widgets/preferences_window.rs:183
+msgctxt ""
+"Name of the color set from the GNOME color palette (https://developer.gnome."
+"org/hig/reference/palette.html)"
+msgid "GNOME color palette"
+msgstr "barevná paleta GNOME"
+
+#: src/widgets/preferences_window.rs:184
+msgid "Show color names from the GNOME color palette"
+msgstr "Zobrazit názvy barev z barevné palety GNOME"
+
+#: src/widgets/preferences_window.rs:188
+msgctxt "Name of the color set from the xkcd color survey"
+msgid "xkcd"
+msgstr "xkcd"
+
+#: src/widgets/preferences_window.rs:189
+msgid "Show color names from the xkcd color survey"
+msgstr "Zobrazit názvy barev z xkcd barevného průzkumu"
+
+#: src/widgets/preferences_window.rs:328
+msgid "Move Up"
+msgstr "Posunout Nahoru"
+
+#: src/widgets/preferences_window.rs:330
+msgid "Move Down"
+msgstr "Posunout Dolu"
+
+#: src/widgets/preferences_window.rs:453
+msgid "Hex-Code"
+msgstr "Hex-Kód"
+
+#: src/widgets/preferences_window.rs:458
+msgid "RGB"
+msgstr "RGB"
+
+#: src/widgets/preferences_window.rs:461
+msgid "HSL"
+msgstr "HSL"
+
+#: src/widgets/preferences_window.rs:464
+msgid "HSV"
+msgstr "HSV"
+
+#: src/widgets/preferences_window.rs:468
+msgid "CMYK"
+msgstr "CMYK"
+
+#: src/widgets/preferences_window.rs:473
+msgid "XYZ"
+msgstr "XYZ"
+
+#: src/widgets/preferences_window.rs:477
+msgid "CIELAB"
+msgstr "CIELAB"
+
+#: src/widgets/preferences_window.rs:482
+msgid "HWB"
+msgstr "HWB"
+
+#: src/widgets/preferences_window.rs:486
+msgid "CIELCh / HCL"
+msgstr "CIELCh / HCL"
+
+#: src/widgets/preferences_window.rs:492
+msgid "Name"
+msgstr "Název"
+
+#: src/widgets/preferences_window.rs:495 src/window.rs:439 src/window.rs:462
+#: src/window.rs:632 src/window.rs:709
+msgctxt "Information that no name for the color could be found"
+msgid "Not named"
+msgstr "Nepojmenováno"
+
+#: src/widgets/preferences_window.rs:500
+msgid "LMS"
+msgstr "LMS"
+
+#: src/widgets/preferences_window.rs:504
+msgid "Hunter Lab"
+msgstr "Hunter Lab"
+
+#: src/window.rs:213
+msgid "Cleared history"
+msgstr "Historie vyčištěna"
+
+#: src/window.rs:214
+msgid "Undo"
+msgstr "Zpět"
+
+#: src/window.rs:535
+msgid "Failed to get palette color"
+msgstr "Získání barevné palety selhalo"
+
+#: src/window.rs:567
+msgid "Failed to pick a color"
+msgstr "Vybrání barvy selhalo"
+
+#. Translators: Do not replace the {}. These are used as placeholders for the copied values
+#: src/window.rs:649
+msgid "Copied: “{}”"
+msgstr "Kopírováno: \"{}\""
```

Comment:
This should use the quotation marks specified in the  [GNOME HIG](https://developer.gnome.org/hig/guidelines/typography.html).
```suggestion
msgstr "Kopírováno: “{}”"
```